src/Analyzers/MSTest.Analyzers/DynamicDataShouldBeValidAnalyzer.cs

```diff
@@ -275,13 +275,17 @@ private static void AnalyzeDataSource(SymbolAnalysisContext context, AttributeDa
 
         // Validate member return type.
         ITypeSymbol? memberTypeSymbol = member.GetMemberType();
-        if (memberTypeSymbol is INamedTypeSymbol memberNamedType
-            && (!SymbolEqualityComparer.Default.Equals(memberNamedType.ConstructedFrom, ienumerableTypeSymbol)
```

Comment:
I think the `ienumerableTypeSymbol` parameter is no longer used and can be cleaned up